options.go

```diff
@@ -0,0 +1,453 @@
+//
+// Copyright (c) 2025 The go-yaml Project Contributors
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package yaml
+
+import (
+	"errors"
+
+	"go.yaml.in/yaml/v4/internal/libyaml"
+)
+
+// LineBreak represents the line ending style for YAML output.
+type LineBreak = libyaml.LineBreak
+
+// Line break constants for different platforms.
+const (
+	LNBreak   = libyaml.LN_BREAK   // Unix-style \n (default)
+	CRBreak   = libyaml.CR_BREAK   // Old Mac-style \r
+	CRLNBreak = libyaml.CRLN_BREAK // Windows-style \r\n
```

Comment:
Usually constants from same type are sharing a name prefix, here they are sharing a suffix.

Having a common prefix helps to find them regrouped in the documentation and IDE completion

Here a common prefix could be LineBreak